share/locale/CMakeLists.txt

```diff
@@ -21,7 +21,8 @@ install(DIRECTORY ./
     DESTINATION ${Mscore_SHARE_NAME}${Mscore_INSTALL_NAME}locale
     FILES_MATCHING
     REGEX ".*\\.qm"
-    REGEX "languages\\.xml")
+    REGEX "languages\\.xml"
+    REGEX ".*_en\\.qm" EXCLUDE)
```

Comment:
Do not install the 'source' language, those should get taken from the compiled in strings